Upstream: no, seems rather inactive
Source: https://build.opensuse.org/projects/multimedia:libs/packages/libcuefile/files/libcuefile-gcc14.patch?expand=1
Reason: Fix build with gcc 14

Index: src/cue_parse.c
===================================================================
--- src/cue_parse.c.orig
+++ src/cue_parse.c
@@ -1508,6 +1508,8 @@ extern int cue_lineno;
 extern int yydebug;
 extern FILE *cue_yyin;
 
+void cue_delete_buffer();
+
 void yyerror (char *s)
 {
 	fprintf(stderr, "%d: %s\n", cue_lineno, s);

